src/frontend/shortlist/src/components/preferences/PreferenceDropdown.vue

```diff
@@ -0,0 +1,62 @@
+<script>
+export default {
+  name: "PreferenceDropdown",
+  props: [
+    "question",
+    "modelValue",
+  ],
+  emits: ["update:modelValue"],
+  computed: {
+    value: {
+      get() {
+        return this.modelValue
+      },
+      set(value) {
+        this.$emit("update:modelValue", value)
+      }
+    }
+  }
+}
+</script>
+
+<template>
+  <div class="pref-q-container">
+    <div class="pref-q-name">{{question.Name}}</div>
+    <div class="pref-q-question">{{question.Text}}</div>
+    <div class="pref-opt-label">
+      <select :name="question.Name" :id="question.Name" v-model="value"
```

Comment:
These lines get super long, so if anyone has any code style suggestions I think we should try to make these as readable as possible (for future PRs)